tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -101,7 +103,7 @@ describe('Test GitRepo Bundle name validation and max character trimming behavio
 });
 
 describe('Test resource behavior after deleting GitRepo using keepResources option', { tags: '@p1'}, () => {
-  const repoName = "local-cluster-fleet-71"
+  const repoName = "local-cluster-fleet-69-70"
```

Comment:
Question: I noticed that if not harcoded here, the second test fails; is this expected behavior?

Reply:
Let me check that again, I don't see any failure due to this.

Reply:
Discussed offline.
Goal is to use dinamic repoName: `const repoName = `local-cluster-fleet-${qase_id}``

Reply:
Okay I will update it.